src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -6721,9 +6721,6 @@ def test_proximity_placement_group(self, resource_group, resource_group_location
             'loc': resource_group_location
         })
 
-        # fails because not a valid type
-        self.cmd('ppg create -n fail_ppg -g {rg} -t notAvalidType', expect_failure=True)
```

Comment:
May I ask why do you remove it?

Reply:
`get_enum_type(self.get_models('ProximityPlacementGroupType'))` will throw an error while passing the parameter instead of after executing the command，`expect_failure` will take effect after executing the command.